ProteinCartography/esmfold_apiquery.py

```diff
@@ -123,7 +124,12 @@ def main():
 
     input_file = args.input
     output_file = args.output
-    esmfold_apiquery(input_file, output_file)
+
+    #Ignore warnings when we make the ESMFold API request
```

Comment:
nit: if you could leave a space between # and the comments, that'd be great, thank you!